frontend/Locales/de.json

```diff
@@ -42,7 +42,7 @@
       "searchingProfile": "Durchsuche Profil",
       "foundProfile": "Profil gefunden",
       "foundContacts": "{{contactsCount}} Kontakte gefunden",
-      "storing": "Storing {{lastEventId}} on database."
+      "storing": "Speichere {{lastEventId}} in Datenbank."
```

Comment:
so fast! 😄 I changed this on a new version I have pending to upload, the new text says:
```
Event {{lastEventId}} stored on database.
```

Reply:
I'll merge for the release you can change it for the next one :)